### ISSUE TITLE
Writerside support

### ISSUE THREAD

**Issue by Ashark** [alex1701c/JetBrainsRunner#41]
The Writerside recent projects are not shown in runner. Can you please add support of it?

**Comment by alex1701c** [comment]
The jetbrains-toolbox installer works for me. Please attach the log file and provide the applications desktop file & the path.

**Comment by Ashark** [comment]
Hm. The log contains recent projects for writerside:
```
Config file found for Writerside /home/andrew/.config/JetBrains/Writerside2023.2/options/recentProjects.xml
===== Recently used project folder for Writerside=====
    /home/andrew/WritersideProjects/untitled
    /home/andrew/Documents/MediaWiki
```
But the runner does not show writerside in the list:

![wrs is not in jbr list](https://github.com/alex1701c/JetBrainsRunner/assets/22634975/e1e9631b-3077-474a-b7f0-93014802262b)

[This](https://aur.archlinux.org/cgit/aur.git/plain/writerside-eap.desktop?h=writerside-eap) is a desktop file used.

**Comment by alex1701c** [comment]
Are the results shown in case you prefix the appname?

**Comment by alex1701c** [comment]
I wonder if it might have sth. to do with the sorting. can you try prefixing the query with "writerside" and check if the runner produces results? Though this is just a guess, because in your screenshot the results are at the top.

Do you have any specific setting for the plugin? Maybe just post a screenshot of the settings window.

**Comment by Ashark** [comment]
Oh. It is strange. But currently the results are shown for it.

![shown results ok](https://github.com/alex1701c/JetBrainsRunner/assets/22634975/f719b337-78b0-4535-bf28-d3e9e13a310c)

What I can think of is:
1 - Maybe it works incorrectly for newly installed apps, and works ok after reboot.
2 - Maybe if it has found the same project opened in another IDE, it thinks it could not be in recents in another IDEs

**Comment by alex1701c** [comment]
> 1 - Maybe it works incorrectly for newly installed apps, and works ok after reboot.

Yeah, that is likely to be true. I have a filewatcher for getting new changes to the recently used xml files, but no mechanism to load new apps.

> 2 - Maybe if it has found the same project opened in another IDE, it thinks it could not be in recents in another IDEs

My initial suspicion too, but the deduplication code works only for the scope of an app.

**Comment by Ashark** [comment]
Ok. I think it is worth adding to the Readme. Other than that, the issue can be closed.

**Comment by alex1701c** [comment]
I will look into fixing the underlying issue :)
